From 9e5c1f4e20dfb57eaa0140dc94ff531511e9617f Mon Sep 17 00:00:00 2001
From: Roger Monk <r-monk@ti.com>
Date: Tue, 2 Mar 2010 14:00:25 +0000
Subject: [PATCH] da850 regulator: Protect function prototypes with CONFIG

   * Regulator usage functions are encapsulated in CONFIG_REGULATOR
   * Function prototypes need the same protection

Signed-off-by: Roger Monk <r-monk@ti.com>
---
 arch/arm/mach-davinci/da850.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-davinci/da850.c b/arch/arm/mach-davinci/da850.c
index 65e8a26..cfecd3c 100755
--- a/arch/arm/mach-davinci/da850.c
+++ b/arch/arm/mach-davinci/da850.c
@@ -1126,8 +1126,10 @@ static struct cpufreq_frequency_table da850_freq_table[] = {
 	},
 };
 
+#ifdef CONFIG_REGULATOR
 static int da850_set_voltage(unsigned int index);
 static int da850_regulator_init(void);
+#endif
 
 static struct davinci_cpufreq_config cpufreq_info = {
 	.freq_table = &da850_freq_table[0],
-- 
1.6.0.6

